brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/submitphoto/SubmitPhotoViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.fabriik.kyc.ui.features.submitphoto
+
+import android.app.Application
+import com.fabriik.common.ui.base.FabriikViewModel
+
+class SubmitPhotoViewModel(
+    application: Application
+
+) : FabriikViewModel<SubmitPhotoContract.State, SubmitPhotoContract.Event, SubmitPhotoContract.Effect>(
+    application
+) {
+
```

Comment:
override `parseArguments` method like this:

override fun parseArguments(savedStateHandle: SavedStateHandle) {
        arguments = SubmitPhotoFragmentArgs.fromBundle(
            savedStateHandle.toBundle()
        )
    }